otterdog/eclipse-ankaios.jsonnet

```
@@ -91,5 +91,32 @@ orgs.newOrg('eclipse-ankaios') {
         },
       ],
     },
+    orgs.newRepo('ank-sdk-rust') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "Eclipse Ankaios Rust SDK provides a convenient rust interface for interacting with the Ankaios platform.",
+      gh_pages_build_type: "legacy",
+      gh_pages_source_branch: "gh-pages",
+      gh_pages_source_path: "/",
+      has_discussions: true,
+      homepage: "https://eclipse-ankaios.github.io/ank-sdk-rust/",
+      topics+: [
+        "automotive",
+        "containers",
+        "orchestration",
+        "ankaios",
+        "sdk",
```

Should we add Rust here:
```suggestion
        "sdk",
        "rust",
```

---

Fixed